qstnrs/qstnrs/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import patterns, include, url
+from django.contrib import admin
+
+admin.autodiscover()
+
+urlpatterns = patterns(
+    '',
+    url(r'^qstnrs/', include('questionnaire.urls')),
+    url(r'^admin/', include(admin.site.urls)),
```

Comment:
I think it's better to allow projects that use this app to make the decision where they want the django admin urls to live. Since the decision weather the django admin app is installed lies in the django project I think it's better to make the urls live there too.